products/knownprojects/bash/03_aggregate.sh

```diff
@@ -28,10 +28,10 @@ echo "SCA sggregations are complete"
 ## Do other aggregation
 # Aggregate KPDB projects to Community Districts
```

Comment:
It seems like the code below creates different level aggregations at census tract, community district, and nta. If that's true, I would revise the comment above and the echo cmd below

Reply:
revised!